src/common/SubProcess.cc

```diff
@@ -49,6 +49,11 @@ void SubProcess::add_cmd_args(const char *arg, ...) {
 void SubProcess::add_cmd_arg(const char *arg) {
   ceph_assert(!is_spawned());
 
+  #ifdef _WIN32
+  if (!arg)
```

Comment:
Is someone passing a `nullptr` to this method? Can it just be avoided at the source (or can we remove the `#ifdef _WIN32`)?

Reply:
I was mostly concerned that we may end up passing the trailing ``nullptr`` from calls such as SubProcess::add_cmd_args but I think we can drop this check.